package.json

```diff
@@ -366,6 +366,11 @@
           "type": "boolean",
           "default": true,
           "description": "Specifes whether OmniSharp should use VS Code editor settings for C# code formatting (use of tabs, indentation size)."
+        },
+        "omnisharp.format.enable": {
```

Comment:
If we want this option to be shared among other extensions that target C#, we should submit it to our barebones csharp extension that is included in VS Code here: https://github.com/Microsoft/vscode/blob/master/extensions/csharp/package.json